plugins/devlink/fu-devlink-netlink.c

```diff
@@ -323,6 +461,21 @@ fu_devlink_netlink_gen_socket_open(GError **error)
 
 	/* initialize structure with properly aligned buffer */
 	nlg->buf = g_malloc0(FU_DEVLINK_NETLINK_BUF_SIZE);
+	nlg->device = device;
```

Comment:
i think a recount here would be a good idea.

Reply:
done